android/src/main/java/com/doublesymmetry/trackplayer/service/MusicService.kt

```diff
@@ -19,16 +19,18 @@ import com.doublesymmetry.trackplayer.utils.Utils.setRating
 import com.facebook.react.HeadlessJsTaskService
 import com.facebook.react.bridge.Arguments
 import com.facebook.react.bridge.Promise
+import com.facebook.react.bridge.WritableMap
```

Comment:
some changes regarding android but I offer them as a patch instead:
[0001-Android-changes.patch.zip](https://github.com/doublesymmetry/react-native-track-player/files/8535075/0001-Android-changes.patch.zip)

Reply:
@jspizziri according to the docs, we should actually not use this experimental `Duration` object, so let's stick to the longs you had before in my patch